openedx/core/djangoapps/oauth_dispatch/toggles.py

```diff
@@ -0,0 +1,9 @@
+"""
+Feature toggle code for oauth_dispatch.
+"""
+
+from openedx.core.djangoapps.waffle_utils import WaffleSwitchNamespace, WaffleSwitch
+
+WAFFLE_NAMESPACE = 'oauth2'
+OAUTH2_SWITCHES = WaffleSwitchNamespace(name=WAFFLE_NAMESPACE)
+UNEXPIRED_RESTRICTED_APPLICATIONS = WaffleSwitch(OAUTH2_SWITCHES, 'unexpired_restricted_applications')
```

Comment:
Do we need 2 different waffle toggles?  Or is the `oauth2.enforce_jwt_scopes` toggle that is defined in drf-extensions sufficient?

Reply:
Yeah, a single switch should do. I will update.